test/uk/gov/hmrc/taxenrolmentassignmentfrontend/reporting/AuditHandlerSpec.scala

```diff
@@ -17,29 +17,34 @@
 package uk.gov.hmrc.taxenrolmentassignmentfrontend.reporting
 
 import org.scalamock.matchers.ArgCapture.CaptureOne
-import org.scalatest.Inside
+import play.api.Application
+import play.api.inject.bind
 import play.api.libs.json.{JsObject, JsString}
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.audit.http.connector.{AuditConnector, AuditResult}
 import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.config.AppConfig
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.helpers.TestFixture
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.helpers.BaseSpec
 
 import scala.concurrent.{ExecutionContext, Future}
 
-class AuditHandlerSpec extends TestFixture with Inside {
+class AuditHandlerSpec extends BaseSpec {
 
   val appname = "appname"
   val auditType = "type"
   val transactionName = "txName"
 
   val mockAuditConnector: AuditConnector = mock[AuditConnector]
 
-  val appConfAuditTest = new AppConfig(servicesConfig) {
-    override lazy val appName: String = appname
-  }
+  override implicit lazy val app: Application = localGuiceApplicationBuilder()
+    .overrides(
+      bind[AuditConnector].toInstance(mockAuditConnector)
+    )
+    .configure(
+      "appName" -> appname
```

Comment:
Why do we need to create a new val for `appname` to assert below, rather than just pulling it from AppConfig?

Reply:
no idea.